src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -238,7 +251,8 @@
              (every? #(isa? % Number) [(fst json-column) (snd json-column)])
              [json-column java.lang.Number]
 
-             (every? #{java.lang.String java.lang.Long java.lang.Integer java.lang.Double java.lang.Boolean}
+             (every? #{java.lang.String java.lang.Long java.lang.Integer
+                       java.lang.Double java.lang.Boolean java.time.LocalDateTime}
```

Comment:
Are you sure it's not better to do something like 

```clj
(every?
 (fn [column-type]
   (some (fn [allowed-type]
           (isa? column-type allowed-type))
         [String Number Boolean java.time.LocalDateTime]))
 [(fst json-column) (snd json-column)])
```

for the same reasons as I mentioned not comparing `(= x String)` above? This isn't an exhaustive list of `Number` subclasses... `BigInteger` and `BigDecimal` are missing, but wouldn't you want to allow those here as well?

Reply:
Also like I mentioned earlier you don't need to put `java.lang.` since those classes are already imported by default

Reply:
I'm almost 99% sure that bignums aren't a thing in json, but could do anyways

Reply:
IIRC JSON doesn't specify the maximum precision of numbers, so it's up to the implementation if it wants to take a number that wouldn't fit in a 64-bit double and return whatever the equivalent of a BigInteger/BigDecimal is instead. I don't know if Postgres is doing this or not but if it's not there's always the chance it might in the future